masterlist.yaml

```diff
@@ -16617,10 +16617,16 @@ plugins:
         condition: 'active("Better Cities Full.esp")'
 
   - name: 'BrotherhoodRenewed.esp'
-    url: [ 'https://www.nexusmods.com/oblivion/mods/31773' ]
+    url:
+      - 'https://www.afkmods.com/index.php?/files/file/260-a-brotherhood-renewed/'
+      - 'https://tesalliance.org/forums/index.php?/files/file/422-a-brotherhood-renewed/'
     tag:
       - Actors.AIPackages
       - Factions
+      - Text
```

Comment:
**Actors.AIPackages** & **Factions** can be removed since they're already in the plugin's description. Looks good otherwise.

```suggestion
    tag: [ Text ]
```